Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'StoryShots/1.9.72 (Linux;Android 9) ExoPlayerLib/2.11.4'
+  os:
+    name: Android
+    version: "9"
+    platform: ""
+  client:
+    type: mobile app
+    name: StoryShots
+    version: "1.9.72"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'Swinsian/472 CFNetwork/978.0.7 Darwin/18.7.0 (x86_64)'
+  os:
+    name: Mac
+    version: "10.14"
+    platform: x64
+  client:
+    type: mobile app
+    name: Swinsian
+    version: "472"
+  device:
+    type: desktop
+    brand: Apple
+    model: ''
+  os_family: Mac
+  browser_family: Unknown
+
+-
+  user_agent: 'ThePodcastApp/6.49.0 (iPhone; iOS 16.5.1; ) player (build 6775;  https://podcast.app/)'
+  os:
+    name: iOS
+    version: "16.5.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: The Podcast App
+    version: "6.49.0"
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'TREBLE/1.0'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Treble
+    version: "1.0"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'TREBLE/2.1'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Treble
+    version: "2.1"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Turtlecast/0.7.7; (Android 13); +https://www.turtlecast.app'
+  os:
+    name: Android
+    version: "13"
+    platform: ""
+  client:
+    type: mobile app
+    name: Turtlecast
+    version: "0.7.7"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'TURTLECAST/0.5.5'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Turtlecast
+    version: "0.5.5"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Ubook Player'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Ubook Player
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'VictorReader_4-4-2'
+  os: [ ]
+  client:
+    type: mobile app
+    name: VictorReader
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'VictorReader Stream 503 ca-en Heather Ryan 4.8.15'
+  os: [ ]
+  client:
+    type: mobile app
+    name: VictorReader
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'VictorReader Stream Trek ca-en Sharon Ryan 2.1.31'
+  os: [ ]
+  client:
+    type: mobile app
+    name: VictorReader
```

Comment:
I believe this is referencing "Victor Reader" product from [Humanware](https://support.humanware.com/en-europe/support/victor_reader_stream_2/victor_reader_stream_2_software). Based on version number and I would guess this is is "Victor Reader Trek"

Reply:
this would probably need a separate regex, as well.